recipe/meta.yaml

```
@@ -1,4 +1,4 @@
-{% set version = "22.13.0" %}
+{% set version = "23.7.0" %}
 
 # NODE_MODULE_VERSION set in src/node_version.h
 {% set NODE_MODULE_VERSION = 127 %}
```

Looks like [`NODE_MODULE_VERSION` was bumped to `131` in the 23.7.0 release]( https://github.com/nodejs/node/blob/73b4c04eb67ee75e9059c536b00939f86f34d582/src/node_version.h#L98 )

```suggestion
{% set NODE_MODULE_VERSION = 131 %}
```

Updating this should fix an existence test failure below